cellcounter/cc_kapi/defaults.py

```diff
@@ -2,12 +2,36 @@
 
 from pytz import utc
 
-DEFAULT_KEYBOARD_MAP = {
+from .models import Keyboard
+
+BUILTIN_DESKTOP_KEYBOARD_MAP = {
+                        "label": u"Default desktop",
```

Comment:
I think you need to include `"user": None` in your serialised default keyboard, akin to the `BUILTIN_MOBILE_KEYBOARD_MAP`, below.